docs/genealignmentsynteny.md

```diff
@@ -0,0 +1,213 @@
+
+```bash
+for i in *.fasta.gz; do
+gunzip $i;
+python3 GA_data_prep.py ${i/.gz} ncbi 10;
+done
+```
+
+This python script command here, in english, means. Take the file, uncompressed, that I downloaded from ncbi and cut it into pieces. A fasta file looks something like this, with headers (lines starting with `>`) and sequence (The usual ATGC's):
+
+```markdown
+> SCAFFOLD_1_DATA_ON_METHODS
+> ATGCGCATGCATGCATCGACTTCGAGCATCGTAG
+> SCAFFOLD_2_DATA_ON_METHODS
+> ACCAGTGCTAGCTAGCTACGTGTGGGTTTGCCCCGTTT
+```
+
+The headers here will be trimmed, to only essential data that you need in order to fine the sequence in your database of choice.
+
+Fasta file may be made up of anywhere between 10's to many thousands of these. So in the case of our cdna and pep files they need to be cut up to let TreeVal have a chance in reading them all in a small time frame.
+
+cds and rna files will be cut up into 1000 header, sequence pairs per file.
+pep and cdna will be cut up by a number you give or by 100.
+
+This is because the size of pep and cdna files are so much larger.
+
+The smaller the number you chunk a file, the smaller the files you produce which means you will also make many more files so there is a trade off.
+
+So will produce a large amount of output in your terminal. Looking like:
+
+```bash
+python3 ../scripts/GA_data_prep.py GallusGallus-GRCg7b.cds.fasta ncbi 100
+
+Your using at least Version 3.6, You are good to go...
+os imported
+argparse imported
+regex imported
+WORKING ON:             cds--GallusGallus-GRCg7b
+Records per file:       1000
+Entryfunction called
+GallusGallus-GRCg7b.cds.fasta
+File found at %s GallusGallus-GRCg7b.cds.fasta
+Renaming headers
+Read_fasta called
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus1000cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus2001cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus3002cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus4003cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus5004cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus6005cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus7006cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus8007cds.MOD.fa
+File saved: -- ./GallusGallus/GallusGallus.GRCg7b/cds/GallusGallus9008cds.MOD.fa
+```
+
+This is pretty much telling us that, yes you have given me a file and for every 1000 (i'm ignoring the number you gave me because this isn't a pep or cdna file) header, sequence pairs I have come across I have made a new file found here. You'll notice that it has also generated a new set of folders. This is based off of how we have named the file.
```

Comment:
```suggestion
This is pretty much telling us that, yes you have given me a file and for every 1000 (I'm ignoring the number you gave me because this isn't a `pep` or `cdna` file) header-sequence pairs I have come across I have made a new file found here. You'll notice that it has also generated a new set of folders. This is based off of how we have named the file.
```

Reply:
Done